### PR TITLE
Introduce rails_admin

### PR DESCRIPTION
**Pull request by lukasbischof**
I did not link SysAdmin to User since to me in this context a user is somebody who uses the product and not somebody how administrates it. Also, some fields are not necessary like an address

### CODE REVIEW

File: vendor/assets/stylesheets/rails_admin/themes/myzivi_theme/theming.scss [hunk 1]
```diff
@@ -0,0 +1,9 @@
+@import './theming/layout';
+@import './theming/content';
+@import './theming/headers';
+@import './theming/tables';
+@import './theming/stats';
+@import './theming/progress';
+@import './theming/list';
+@import './theming/buttons';
+@import './theming/forms';
```
Comment (lukasbischof) [inline vendor/assets/stylesheets/rails_admin/themes/myzivi_theme/theming.scss:9]: All those files in `vendor/assets/stylesheets/rails_admin/themes` were copied from a different theme and adapted a bit with myzivi style

File: vendor/assets/stylesheets/rails_admin/themes/myzivi_theme/mixins.scss [hunk 2]
```diff
@@ -0,0 +1,8 @@
+/*
+  Customize Sass mixins from Twitter-Bootstrap/RailsAdmin theme or add new ones for your own use.
+
+  Available mixins to use/override:
+
+  https://github.com/twbs/bootstrap-sass/tree/master/assets/stylesheets/bootstrap/mixins
+  https://github.com/sferik/rails_admin/blob/master/app/assets/stylesheets/rails_admin/base/mixins.scss
+*/
```
Comment (TheFehr) [inline vendor/assets/stylesheets/rails_admin/themes/myzivi_theme/mixins.scss:8]: ?

Comment (lukasbischof) [reply vendor/assets/stylesheets/rails_admin/themes/myzivi_theme/mixins.scss:8]: This came from theme I copied

Comment (lukasbischof) [reply vendor/assets/stylesheets/rails_admin/themes/myzivi_theme/mixins.scss:8]: But I can delete it probably

**Review by TheFehr — APPROVED** [review]
Besides the usless file, looks good. Looking forward to having a nice admin panel

**Pull request merged**